JSON-Schema/schemas/definitions-JSON-Schema.json

```diff
@@ -35,6 +35,22 @@
         "required-ld-context": {
             "type": "object",
             "required": ["@context"]
+        },
+        "version": {
+            "type": "string",
+            "pattern":  "^\\d+(\\.\\d+)?$"
+        },
+        "id": {
+            "oneOf": [
+                {
+                    "type": "string",
+                    "format": "uri"
+                },
+                {
+                    "type": "string",
+                    "pattern": "_:([a-z]|[A-Z]|[0-9]|%)*"
```

Comment:
@mgh128 this is the pattern I propose for blank nodes in EPCIS 2.0 JSON-LD documents

Reply:
Thanks - that regular expression for the blank node pattern looks reasonable.  They are usually either alphanumeric or hexadecimal in the tools that auto-generate them.  Just wondering whether we need to also support a hyphen in case any tool is using a UUID for the part that follows the _:  .  If we add support for a literal hyphen, then UUIDs are also supported.

Reply:
that's perfectly fine, adding it

Reply:
Hi @jmcanterafonseca-iota  - I'm having some further thoughts about blank nodes.  The notation _:123abc etc. is usually used in RDF Turtle but typically not used within JSON-LD, in which a blank node is typically expressed via nesting of an object that doesn't declare an "@id" (or something that is an alias of "@id") - so I think we might need to rethink that or test it with some examples using JSON-LD playground.  For example, at https://json-ld.org/playground/#startTab=tab-nquads&json-ld=%7B%22%40context%22%3A%22http%3A%2F%2Fschema.org%2F%22%2C%22%40type%22%3A%22Person%22%2C%22name%22%3A%22John%20Smith%22%2C%22address%22%3A%7B%22%40id%22%3A%22_%3AmyBlankNodeIdentifier%22%2C%22postalCode%22%3A%22CB2%203BU%22%7D%7D

you can see that even if we use "@id" to specify a custom blank node identifier, when that JSON-LD is converted to RDF Turtle, the custom blank node identifier need not be preserved in the RDF triples that are extracted.

So, if you're preparing this for a property that is an alias of "@id" and expects an IRI/URI and acts as the RDF Subject of some triples, then I think it is probably better to make that property optional, rather than use JSON Schema to tolerate a Turtle-format blank node identifier as an alternative to a URI.

Reply:
blank node identifiers only make sense to be used as internal anchors within a JSON-LD document 
We have blank node identifiers in the majority of EPCISDocument for the `id` property , I am wondering whether we should convert those blank node identifiers to be just example.org URIs ... i.e. an EPCIS Document or QueryDocument may include an `id` but it shall be a complete URI and not a blank node identifier

Reply:
OK - good point regarding the examples.  Yes, we should change those to example.org URIs and avoid using any Turtle-style blank node identifiers in JSON-LD examples.  My mistake in the examples several months ago.

Reply:
now fixed. it seems that auto-replace prettified on its own some of the JSON files. Let me know if that's ok @mgh128